doc/sentence_length_diffs_inference_report.Rmd

```diff
@@ -25,85 +25,86 @@ ci_95 <- readRDS(paste(params$results_dir_path,"/ci_95.rds", sep=""))
 diff_medians <- readRDS(paste(params$results_dir_path, "/diff_medians.rds",  sep=""))
 ```
 
-# Summary 
+# Summary
+
+For this project we have carried out a hypothesis test to determine if there was a significant difference in the median sentence lengths between the indigenous and non-indigenous offenders under the Correction Services Canada.
 
-For this project we have carried out a hypothesis test to determine if there was a significant difference in the median sentence lengths between the indigenous and non-indigenous offenders under the Correction Services Canada. The median was selected as the measure of central tendency as the distribution of sentence length for both groups were skewed. Subsequently, a permutation test under the null model was carried out computationally with a significance level of 0.05. The null hypothesis stated that there was no difference in the population medians in sentence length between indigenous and non-indigenous offenders. The alternate hypothesis stated that there is a difference in the population medians in sentence length between indigenous and non-indigenous offenders. The resulting sample difference in the two medians was `r diff_medians`, with a corresponding p-value of `r p_value`. The indigenous group was found to have shorter sentence lengths than the non-indigenous group. As this $p$-vaule was smaller than the significance level, there was statistically significant evidence to reject the null hypothesis that stated that there was no difference in the median sentence lengths between the two groups. We had a large sample size for both groups, so the model was very sensitive to small differences in the median variation. Though this may raise some concern regarding the practical implications of the study, we believed it was important not to miss any existing effect due to the sensitivity of the issue at hand. The cost of a Type II error is more significant than a Type I error.   
+The median was selected as the measure of central tendency as the distribution of sentence length for both groups were skewed. Subsequently, a permutation test under the null model was carried out computationally with a significance level of **0.05**. The null hypothesis stated that there was no difference in the population medians in sentence lengths between indigenous and non-indigenous inmates. The alternative hypothesis stated that there was a difference in the population medians in sentence length between indigenous and non-indigenous offenders. The resulting sample difference in the two medians was `r diff_medians`, with a corresponding p-value of `r p_value`. The indigenous group was found to have shorter sentence lengths than the non-indigenous group. As this $p$-vaule was smaller than the significance level, there was statistically significant evidence to reject the null hypothesis that stated that there was no difference in the median sentence lengths between the two groups.
```

Comment:
Should we be a little more specific and perhaps say: 
The median was selected as the measure of central tendency as the distribution of sentence length for both groups were _right_ skewed.